src/include/kompute/logger/Logger.hpp

```diff
@@ -25,6 +25,7 @@ static const char* KOMPUTE_LOG_TAG = "KomputeLog";
 #else
 #if KOMPUTE_BUILD_PYTHON
 #include <pybind11/pybind11.h>
+#include <fmt/core.h>
```

Comment:
You could also do:

```c++
#if KOMPUTE_BUILD_PYTHON
#include <pybind11/pybind11.h>
namespace py = pybind11;
// from python/src/main.cpp
extern py::object kp_trace, kp_debug, kp_info, kp_warning, kp_error;
#endif // KOMPUTE_BUILD_PYTHON
#include <fmt/core.h>
#endif // VK_USE_PLATFORM_ANDROID_KHR
```

As in the code in the PR you have #include <fmt/core.h> in the #if and the #else.